README.md

```diff
@@ -25,6 +25,7 @@ Proxy for handling the cache for pages within the legacy CMS
 | BABBAGE_URL                  | http://localhost:8080  | Babbage address, where all the incoming requests are forwarded to                                                                    |
 | LEGACY_CACHE_API_URL         | http://localhost:29100 | Legacy Cache API address                                                                                                             |
 | RELEASE_CALENDAR_URL         | http://localhost:27700 | Release calendar frontend controller address                                                                                         |
+| ENABLE_RELEASE_CALENDAR      | false                  | Flag to enable `/releases/{uri:.*}` URLs to go through dp-frontend-release-calendar instead.                                         |
```

Comment:
Flag to enable incoming requests to be forwarded to dp-frontend-release-calendar.